<?xml version="1.0"?>
<robot name="lio" xmlns:xacro="http://tixiaoshan.github.io/">
  <xacro:property name="PI" value="3.1415926535897931" />
  <link name="map"></link>
  <link name="odom"></link>
  <joint name="map_joint" type="fixed">
    <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
    <parent link="map"/>
    <child link="odom"/>
  </joint>

  <link name="base_link"></link>
  <joint name="odom_joint" type="continuous">
    <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
    <parent link="odom"/>
    <child link="base_link"/>
  </joint>

  <link name="imu_link"> </link>
  <joint name="imu_joint" type="fixed">
    <parent link="base_link" />
    <child link="imu_link" />
    <origin xyz="0 0 0" rpy="0 0 0" />
  </joint>

  <link name="navsat_link"> </link>
  <joint name="navsat_joint" type="fixed">
    <parent link="base_link" />
    <child link="navsat_link" />
    <origin xyz="0 0 0" rpy="0 0 0" />
  </joint>

</robot>
